plaso/cli/tools.py

```diff
@@ -507,3 +511,38 @@ def Write(self, string):
       # sys.stdout.write() on Python 3 by default will error if string is
       # of type bytes.
       sys.stdout.write(string)
+
+
+class FindSpecsGetter(object):
+  """Get FindSpecs for CLI Tools."""
+
+  def GetFindSpecs(
+      self, artifacts_registry, artifact_filter_names, filter_file_path,
+      knowledge_base):
+    """Get Find Specs from artifacts or filter file if available.
+
+    Args:
+       artifact_filters (str): Path to file listing artifact filters by
+          name or artifact names listed directly, comma separated.
+       filter_file_path (str): Path of filter file.
+       knowledge_base (KnowledgeBase): Knowledge base.
+    """
+    if artifact_filter_names and filter_file_path:
+      raise RuntimeError('Please only specify one type of filter file, '
```

Comment:
consistency nit: in the plaso codebase we typically break after `(` and continue with 4 space continuation indentation

Reply:
Done